db/attachment_compaction_test.go

```diff
@@ -921,18 +921,18 @@ func createDocWithInBodyAttachment(t *testing.T, ctx context.Context, docID stri
 
 		attachmentSyncDataBytes, err := base.JSONMarshal(attachmentSyncData)
 		if err != nil {
-			return nil, base.Uint32Ptr(0), false, err
+			return nil, base.Ptr(uint32(0)), false, err
 		}
 
 		updated, err = base.InjectJSONPropertiesFromBytes(current, base.KVPairBytes{
 			Key: "_attachments",
 			Val: attachmentSyncDataBytes,
 		})
 		if err != nil {
-			return nil, base.Uint32Ptr(0), false, err
+			return nil, base.Ptr(uint32(0)), false, err
```

Comment:
this is just test code, change to more sensible API:

```suggestion
			return nil, nil, false, err
```